build.gradle

```diff
@@ -89,24 +89,26 @@ dependencies {
     implementation group: 'ch.qos.logback.contrib', name: 'logback-json-classic', version: '0.1.5'
     implementation group: 'ch.qos.logback.contrib', name: 'logback-jackson', version: '0.1.5'
 
-    // OpenTelemetry
-    var openTelemetryVersion = '1.34.1'
-    var openTelemetryInstrumentationVersion = '2.0.0'
-    implementation "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-sdk:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-sdk-metrics:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-exporter-logging:${openTelemetryVersion}"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-webmvc-6.0:${openTelemetryInstrumentationVersion}-alpha"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-instrumentation-api:${openTelemetryInstrumentationVersion}"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:${openTelemetryInstrumentationVersion}"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-boot:${openTelemetryInstrumentationVersion}-alpha"
-    implementation "io.opentelemetry:opentelemetry-exporter-prometheus:${openTelemetryVersion}-alpha"
-    implementation "io.opentelemetry:opentelemetry-extension-incubator:${openTelemetryVersion}-alpha"
+    // OpenTelemetry BOMs
+    implementation platform('io.opentelemetry:opentelemetry-bom:1.34.1')
+    implementation platform('io.opentelemetry:opentelemetry-bom-alpha:1.34.1-alpha')
```

Comment:
Why do we need the alpha one?

Reply:
Ah nevermind, I read your PR summary.

Reply:
To answer your question more concretely, here's what we previously pulled in that we now get from this Alpha BOM:

```
    implementation "io.opentelemetry:opentelemetry-exporter-prometheus:${openTelemetryVersion}-alpha"
    implementation "io.opentelemetry:opentelemetry-extension-incubator:${openTelemetryVersion}-alpha"
    ```